src/metrics/external-types.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {GoogleAuthOptions} from 'google-auth-library';
+
+export interface ExporterOptions {
+  /**
+   * Optional authentication options for Google services.
+   */
+  authOptions?: GoogleAuthOptions;
+  /**
+   * Prefix prepended to OpenTelemetry metric names when writing to Cloud Monitoring. See
+   * https://cloud.google.com/monitoring/custom-metrics#identifier for more details.
+   *
+   * Optional, default is `workload.googleapis.com`.
+   */
+  prefix?: string;
```

Comment:
This should not be configurable for our use case as metrics will always go to "spanner.googleapis.com"

Reply:
Got it, removed.